README.md

```diff
@@ -1,2 +1,72 @@
-# Ner4Opt
-Ner4Opt: Named-Entity Recognition for Optimization
+# Named Entity Recognition for Optimization (Ner4Opt) Library
+
+Given an optimization problem in natural language, the library extracts **six named entities**:
+
+- **CONST_DIR**: Constraint direction
+- **LIMIT**: Limit
+- **OBJ_DIR**: Objective direction
+- **OBJ_NAME**: Objective name
+- **PARAM**: Parameter
+- **VAR**: Variable
+
+## Installation
+
+To install this package, run the following command from the root folder
+
+```bash
+pip install .
+```
+
+## Quick Start
+
+```python
+# Import the Ner4Opt Library
+from ner4opt import Ner4Opt
+
+# Problem Description
```

Comment:
Here is a good place to add a comment about spacy tokenization.
If you have unspacy free-form text, how would I "spacy-fy" it?

Reply:
Added a line about spacy tokenization.

Reply:
where is that added line? Sorry if i am not seeing sth obvious

Reply:
I went up and down a few times but not sure where is this. may be you are referring to some other bullet points under data folder under the training folder.. A typcialy user is unlikely to understand that.. 

My base question is; what happens when we run this without the spaces? Does it not find the same entities? 

Assuming it does not work with "regular" text that is not spacify-ied, we need to tell the user in the readme how to spacify their input. 

Can we add a quick section below the Installation below, sth like "Input Text Format" and explain briefly what is needed and why. Then, can we give the user a one-liner or sth to "spacify" their text (unless we cannot do this automatically under the hood all the time, whenever the user passes the text to us? On that note; what happens if we always "spacify" under the hood to relieve the user from this detail? OR.. it is not gonna work out if we spacify all the time, in case, the text was already spacify-ied)